src/lib/viewers/doc/ImagePageLoader.js

```diff
@@ -0,0 +1,89 @@
+import EventEmitter from 'events';
+import RepStatus from '../../RepStatus';
+
+const REPS = ['png_1024x1024', 'png_2048x2048', 'jpg_1024x1024'];
+
+class ImagePageLoader extends EventEmitter {
+    /**
+     * [constructor]
+     *
+     * @param {Object} options Viewer options
+     */
+    constructor(options) {
+        super();
+        this.availableReps = {};
+        this.options = options;
+        this.checkAvailableReps();
+    }
+
+    /**
+     * Gets a representation based on preview size
+     *
+     * @param {number} previewHeight - Height of preview window
+     * @param {number} previewWidth - Width of preview window
+     * @return {Object} Instance of RepStatus
+     */
+    getRepresentation(previewHeight, previewWidth) {
```

Comment:
Right now this only considers preview dimensions, but could eventually consider device type, network conditions, zoom level etc.

Reply:
To make this a little more future proof, could you have the parameters be a single object, and then extract `previewHeight` and `previewWidth` from it?